assets/src/components/detours/detourMap.tsx

```diff
@@ -378,6 +393,47 @@ const OriginalRouteShape = ({
   </>
 )
 
+interface UnfinishedDiversionRouteShapeProps {
+  interactive: boolean
+  segments: UnfinishedRouteSegments
+  onClick: (e: LeafletMouseEvent) => void
+}
+
+const UnfinishedDiversionRouteShape = ({
+  interactive,
+  segments: { beforeStartPoint, afterStartPoint },
+  onClick,
+}: UnfinishedDiversionRouteShapeProps) => (
+  <>
+    <Polyline
+      weight={6}
+      interactive={false}
+      positions={beforeStartPoint.map(shapePointToLatLngLiteral)}
+      className="c-detour_map--original-route-shape-core"
+    />
+    <Polyline
+      weight={6}
+      interactive={false}
+      positions={afterStartPoint.map(shapePointToLatLngLiteral)}
+      className="c-detour_map--original-route-shape-core"
+    />
```

Comment:
Could these be easily combined into 1 polyline by passing both segments into positions?

Reply:
non-blocking question:  

Related to the above comment, I just looked at the `OriginalRouteShape` code for the first time, which, naturally, overlaps with this alternative `UnfinishedDiversionRouteShape`. It occurs to me, could they have been a single component? It could still be called `OriginalRouteShape`, as it still references the existing route pre-detour. It would need a new field for the positions of the interactive line, but that's it. Basically, once a detour is starting to be drawn, the interactive line is truncated, but the core line remains the same. 

Not yet asking for a change here (would want to weight the cost-benefit before genuinely suggesting a change)

Reply:
Hummmmmm - I'm wondering if it would make more sense to combine this PR with https://github.com/mbta/skate/pull/2639. A lot of the suggestions you're making.. I think they do make sense in the context of this work alone, but I think I'd have to undo them in order to to https://github.com/mbta/skate/pull/2639

Reply:
But yeah... the "core" could be its own element, since it renders the same in all three of `OriginalRouteShape`, `UnfinishedDiversionRouteShape`, and `DivertedRouteShape`.